Mage/Module.cs

```diff
@@ -126,17 +160,12 @@ public void SetupUsercommands(Action<Exception> cb)
 			commandHandlerActions = new Dictionary<string, Action<JObject, Action<Exception, JToken>>>();
 			commandHandlerFuncs = new Dictionary<string, Func<JObject, UserCommandStatus>>();
 
-			if (Commands == null)
```

Comment:
From what I remember, In the current implementation you can go without defining that list, meaning if your module has no user commands, no need to even touch the variable. This would change that behaviour (If I'm not mistaken)